modules/userId.md

```diff
@@ -0,0 +1,72 @@
+## User ID Example Configuration
+
+Example showing `cookie` storage for user id data for both submodules
+```
+pbjs.setConfig({
+    usersync: {
+        userIds: [{
+            name: "unifiedId",
+            params: {
+                partner: "prebid",
+                url: "http://match.adsrvr.org/track/rid?ttd_pid=prebid&fmt=json"
+            },
+            storage: {
+                type: "cookie",
+                name: "unifiedid",
+                expires: 60
+            }
+        }, {
+            name: "pubCommonId",
+            storage: {
+                type: "cookie",
+                name: "pubcid",
```

Comment:
The original cookie name is _pubcid.  Should that be maintained?

Reply:
This was meant to be an example only, but anyhow updated the example.